bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestCheckAndMutate.java

```diff
@@ -49,20 +51,27 @@ public void testCheckAndPutSameQual() throws Exception {
       byte[] value2 = dataHelper.randomData("value-");
 
       // Put with a bad check on a null value, then try with a good one
-      Put put = new Put(rowKey).addColumn(SharedTestEnvRule.COLUMN_FAMILY, qual, value1);
-      boolean success = checkAndPut(rowKey, SharedTestEnvRule.COLUMN_FAMILY, qual, value2, put);
-      Assert.assertFalse("Column doesn't exist.  Should fail.", success);
-      success = checkAndPut(rowKey, SharedTestEnvRule.COLUMN_FAMILY, qual, null, put);
-      Assert.assertTrue(success);
+      Put put1 = new Put(rowKey).addColumn(SharedTestEnvRule.COLUMN_FAMILY, qual, value1);
+      Assert.assertFalse(
+          "Column doesn't exist. Should fail.",
+          checkAndPut(rowKey, SharedTestEnvRule.COLUMN_FAMILY, qual, value2, put1));
+      Assert.assertFalse(
```

Comment:
same here and below everywhere

Reply:
Converting to use static imports will create a huge diff and make it very hard to review the real fix. It could be done separately and applied to other tests next to this one.